airflow/bin/cli.py

```diff
@@ -259,39 +259,63 @@ def pools(args):
     :param args: parsed args object with a property of filepath to airflow pools yaml file
     :return: None
     """
+    session = settings.Session()
     with open(args.filepath, 'r') as stream:
         try:
             pools_config = yaml.load(stream)
-        except yaml.YAMLError:
+        except yaml.YAMLError as e:
+            print("Failed to load: {}".format(args.filepath))
```

Comment:
If all we're doing is printing the exception, I don't think we need this except block at all. The exception should already get logged if it bubbles up